data/strings.s

```diff
@@ -351,19 +351,19 @@ gUnknown_8416133:: @ 0x8416133
 gUnknown_8416146:: @ 0x8416146
 	.string "ポケモン$"
 
-gUnknown_841614B:: @ 841614B
+gText_SaveError_PleaseExchangeBackupMemory:: @ 841614B
 	.string "Save error.\pPlease exchange the\nbackup memory.$"
```

Comment:
Possibly you can split this into separate lines for consistency with other strings?